packages/salesforce/src/Adaptor.js

```diff
@@ -280,16 +283,10 @@ async function pollJobResult(conn, job, pollInterval, pollTimeout) {
   }
 }
 
-const defaultOptions = {
-  pollTimeout: 90000, // in ms
-  pollInterval: 3000, // in ms
-};
-
 /**
  * Execute an SOQL Bulk Query.
  * This function uses bulk query to efficiently query large data sets and reduce the number of API requests.
- * Note that in an event of a query error,
- * error logs will be printed but the operation will not throw the error.
+ * The `bulkQuery()` uses {@link https://sforce.co/4b6kn6z Bulk API v.2.0} which is available in API version 41.0 and later.
```

Comment:
This links to API limits, which is useful but surprising :thinking:

Reply:
I have updated the links @josephjclark